### DISCUSSION TITLE
AM scan sensitivity?

### DISCUSSION THREAD

**Discussion by Overland-DX** [PE5PVB/TEF6686_ESP32#349] [Q&A]
Is it possible to change the sensitivity of the scan function, so that it can stop on weaker stations?
The way it works now, I'm lucky if it stops on very strong stations.

**Comment by ohmytime** [comment]
Yes, I think it is too high for now.

**Comment by PE5PVB** [comment]
I made it more sensitive. Please check. If not ok, report it as an issue.
